support.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Support
+page_title: Support
+description: Copyright
+slug: support-page
+tags: Fiddler support, trial, pro version, paid support
+published: True
+position: 80
+---
+
+## Fiddler Everywhere Support 
+
+ 
+#### Who has access to support? 
+
+All users who have obtained a Trial subscription or PRO plan subscription will have access to support for the duration of the active subscription period, the table below describes the comparison between different plans and level of support offered for each plan. 
+
+
+<!-- TODO the plan table here -->
+|   |**Free**   |**Trial**   |**Pro**  |
+|---|---|---|---|
+| Email support |  - |  __Unlimited__* | __Unlimited__* |
+| Community Forums   | Yes | Yes | Yes |
+| Feedback Portal   | Yes | Yes | Yes |
+
+*subject to __fair usage policy__
+
+#### What is the Fair Usage Policy? 
+
+Most of our product licenses include unlimited access to the support service. In order to keep providing our industry-leading level of support, we have instituted a fair usage policy for our support services: 
```

Comment:
1) Remove "most", since we have just one at the moment (Trial is not a separate type)
2) Use **subscription** instead of license and stick to that word. **License** has a slightly different meaning and if we want to use it we should stick to it and use it everywhere instead.
For example:
The Fiddler Everywhere subscriptions include...